src/StarkEx.Crypto.SDK/Hashing/SpotTradingMessageHasher.cs

```diff
@@ -141,6 +222,35 @@ public BigInteger EncodeConditionalTransferWithFees(EncodeTransferWithFeesModel
         return pedersenHash.CreateHash(fourthInnerHash, fifthWeight);
     }
 
+    public BigInteger EncodeConditionalTransferWithoutFees(EncodeTransferWithoutFeesModel encodeTransferWithoutFeesModel, string condition)
```

Comment:
What was the reason for overloading the `EncodeConditionalTransferWithoutFees` method to explicitly accept a condition? The `EncodeConditionalTransferWithoutFees` is also being overloaded in the same way.

Reply:
As talked in daily, it was to be able to use the same test dataset from StarkWare which only includes the condition